code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -97,6 +97,19 @@
 				if(T.canBuildPlating(S) == BUILD_SUCCESS)
 					qdel(L)
 
+/obj/item/stack/tile/wood/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	if(iswrench(W))
+		if(amount >= 4)
+			playsound(get_turf(src), 'sound/items/Ratchet.ogg', 50, 1)
+			drop_stack(/obj/item/stack/sheet/wood, get_turf(user), 1, user)
+			use(4)
+			return
```

Comment:
Instead of putting the returns here and at the end of the else clause, put them after the `if`/`else` blocks so there's only.